### ISSUE TITLE
Support Cassandra connector.

### ISSUE THREAD

**Issue by GOODBOY008** [ververica/flink-cdc-connectors#821]
I want to know what about flink-cdc-connectors support Cassandra? Welcome to append your idea in this issue.

**Comment by waywtdcc** [comment]
It's ok

**Comment by DonaldZhang** [comment]
Any fresh developments ?

**Comment by DonaldZhang** [comment]
Yeah, do you have any suggestions ?

**Comment by GOODBOY008** [comment]
> Collaborator

Are you willing to contriute cassandra connector?

**Comment by DonaldZhang** [comment]
Of course. I'm not very familiar with Cassandra. I'm not sure whether it will affect contribution.

**Comment by GOODBOY008** [comment]
> Of course. I'm not very familiar with Cassandra. I'm not sure whether it will affect contribution.

It's ok, We will give help ,if you need.

**Comment by DonaldZhang** [comment]
That's great. I want to know how to start.

**Comment by GOODBOY008** [comment]
> That's great. I want to know how to start.

Clone code and study other connector source code.

**Comment by DonaldZhang** [comment]
OK, I will try it.

**Comment by miguelangelprogramacion** [comment]
Is there any progress in this direction?

**Comment by PatrickRen** [comment]
Closing this issue because it was created before version 2.3.0 (2022-11-10). Please try the latest version of Flink CDC to see if the issue has been resolved. If the issue is still valid, kindly report it on [Apache Jira](https://issues.apache.org/jira) under project `Flink` with component tag `Flink CDC`. Thank you!
